recipes/skll/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "skll" %}
+{% set version = "2.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/EducationalTestingService/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5d690556a72a729b4039d50010ddf7470c68d97505e8b6b6f1a5cc29d2b0ae01
+
+build:
+  noarch: python
+  number: 0
+  script:  
+    - cd $SRC_DIR
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Is there any reason not to do:
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

---

For some reason it wasn't working (maybe I was doing something wrong). Let's try to revert it and see how it goes. I will wait for the other pass to run first...

---

Okay, I think this will work too. Accepting.